package/boot/uboot-mvebu/patches/0003-arm-mvebu-Initial-ESPRESSOBin-Ultra-board-support.patch

```diff
@@ -0,0 +1,516 @@
+From 0a6d2787f8b65977963b21d77eed23d055f19613 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Kerma=20G=C3=A9rald?= <gandalf@gk2.net>
```

Comment:
Same as above. Keep the original author.

Reply:
It is a from… not a copyright